account_product_fiscal_classification/models/account_product_fiscal_classification_model.py

```diff
@@ -21,7 +21,7 @@
 ##############################################################################
```

Comment:
Please switch long headers to [shortened header style](https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3)